scipy/special/basic.py

```diff
@@ -754,20 +754,97 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse error function.
+    This function compute the inverse error function.
+    Inverse error function is satisfying :math:`erf(erf^{-1}(x))=x`.
+
+    Parameters
+    ----------
+    y : ndarray
+        Argument at which to evaluate for (-1, 1)
+
+    Returns
+    -------
+    erfinv : ndarray
+        Value of erfinv(y0), ..., erfinv(yn)
+
+    Examples
+    --------
+    1) evaluating a float number
+
+    >>> from scipy import special
+    >>> special.erfinv(0.5)
+    0.4769362762044698
+
+    2) evaluating a ndarray
+
+    >>> from scipy import special
+    >>> import numpy as np
```

Comment:
import of numpy not needed.

Reply:
In that example case, the result of erfinv is using a numpy array. 
I would like to know the reason that import of numpy is not needed.

Reply:
Because that's the convention in all of the docstrings.

Reply:
OK. I sorry that did not noticed the convension. I will remove import numpy.
Thanks @pv